rules/properties-alphabetical-order/tests/index.js

```diff
@@ -68,6 +68,9 @@ testRule({
 		{
 			code: 'a { font-size: 1px; -moz-osx-font-smoothing: grayscale; -webkit-font-smoothing: antialised; font-weight: bold; }',
 		},
+		{
+			code: 'a { color: #000; padding: { bottom: 1em; top: 1em; } width: 25%;}',
```

Comment:
Make it CSS instead of SCSS:

```suggestion
			code: 'a { color: #000; span { bottom: 1em; top: 1em; } width: 25%;}',
```

Reply:
That's the example from the ticket :/